functions-and-operators/string-functions.md

```diff
@@ -1188,20 +1188,39 @@ Return a string containing octal representation of a number.
 
 ### [`OCTET_LENGTH()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_octet-length)
 
-Synonym for `LENGTH()`.
+Synonym for [`LENGTH()`](#length).
 
 ### [`ORD()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_ord)
 
 Return character code for leftmost character of the argument.
 
 ### [`POSITION()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_position)
 
-Synonym for `LOCATE()`.
+Synonym for [`LOCATE()`](#locate).
 
 ### [`QUOTE()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_quote)
 
 Escape the argument for use in an SQL statement.
 
+If the argument is `NULL`, the function returns `NULL`.
+
+Examples:
+
+The following example shows that the ASCII NULL character is escaped as `\0` and the single quote character `'` is escaped as `\'`:
+
+```sql
+SELECT QUOTE(0x002774657374);
+```
+
+```
++-----------------------+
+| QUOTE(0x002774657374) |
++-----------------------+
+| '\0\'test'            |
++-----------------------+
+1 row in set (0.00 sec)
+```
+
```

Comment:
What is your setup when running this?
When I run this with `tiup playground`, i.e. using TiDB v8.0.0 I get:
```mysql
tidb> SELECT QUOTE(0x002774657374);
+----------------------------------------------+
| QUOTE(0x002774657374)                        |
+----------------------------------------------+
| 0x275C305C277465737427                       |
+----------------------------------------------+
1 row in set (0.01 sec)
```

Maybe something like this example:
```mysql
tidb> SELECT QUOTE("it's quoted!");
+-----------------------+
| QUOTE("it's quoted!") |
+-----------------------+
| 'it\'s quoted!'       |
+-----------------------+
1 row in set (0.00 sec)
```
and an explanation that it also handles other characters like the zero byte etc.

Reply:
I've added a note about `mysql --skip-binary-as-hex ...`

Reply:
@mjonss Do you think this is ok now?

Reply:
> To display the result as a hexadecimal-encoded value instead of showing it directly, you need to start the MySQL client with the [`--skip-binary-as-hex`](https://dev.mysql.com/doc/refman/8.0/en/mysql-command-options.html#option_mysql_binary-as-hex) option.

@dveeden, does not `--skip-binary-as-hex` do the opposite?
I.e. using `--skip-binary-as-hex` will show the string as-is instead of as hex, even if the server considers it to be binary data?

Reply:
Yes indeed